viewPost.php

```diff
@@ -1,31 +1,42 @@
 <?php
 
 include('config/init.php');
+$comment=getBlogPostComments($_GET['blogPostId']);
+$result=getAllCategories($_GET ['blogPostId']);
```

Comment:
Not a huge deal, but I wouldn't call this `$result` where there are other "results" above and below it. Would be better to give it a more specific name.

Although it looks like you're not using it anywhere on this page so maybe you can just remove it entirely?